modules/formulize/class/formulize.php

```diff
@@ -100,3 +99,141 @@ static function serializedDBFields() {
 	}
 
 }
+#[AllowDynamicProperties]
+class formulizeHandler {
```

Comment:
This is not really a handler like the other handlers. But I followed the same structure in terms of the file because that would be consistent? But the methods are just static and accessed directly.

Since they're static, they could just be methods on the formulizeObject. But they do all kinds of things beyond what would just be tools/methods pertaining to a form or an element or a screen, etc. So it seemed like they should be at a different level, so a "formulizeHandler" made sense? Or did it?

Very open to suggestions about where this should live, how this should be organized.